app.py

```diff
@@ -32,39 +32,35 @@ def main():
     comparison_score = flatliners.ComparisonScore()
     corr_comparison_score = flatliners.CorrComparisonScore()
 
+    single_value_metric = flatliners.SingleValueMetric()
+    versioned_metrics.subscribe(single_value_metric)
+
     # one will get versioned metrics
-    versioned_metrics.subscribe(std_dev_cluster)  # take etcd data and perform std_dev_cluster operation
+    # take etcd data and perform std_dev_cluster operation
+    single_value_metric.subscribe(std_dev_cluster)
     std_dev_cluster.subscribe(std_dev_version)
 
     # std_dev_cluster emits the std_dev for a cluster
     # this is something std_dev_version is interested in
-
     std_dev_cluster.subscribe(comparison_score)
     std_dev_version.subscribe(comparison_score)
 
     # Alert correlation
     alert_cor = flatliners.ClusterAlertCorrelation()
-    versioned_metrics.subscribe(alert_cor)
-    # alert_cor.subscribe(print) # this emits a df with the correlation values for a single cluster
+    single_value_metric.subscribe(alert_cor)
 
     # Git version alert correlation
     version_alert_corr = flatliners.GitVersionAlertCorrelation()
-
-    versioned_metrics.subscribe(version_alert_corr)
-
-    # version_alert_corr.subscribe(print) # this emits a df with correlation values for a gitVersion
+    single_value_metric.subscribe(version_alert_corr)
 
     alert_cor.subscribe(corr_comparison_score)
-
     version_alert_corr.subscribe(corr_comparison_score)
 
-    # corr_comparison_score.subscribe(print)
-
     weirdness_score = flatliners.WeirdnessScore()
     comparison_score.subscribe(weirdness_score)
     corr_comparison_score.subscribe(weirdness_score)
 
-    # weirdness_score.subscribe(print)
+    weirdness_score.subscribe(print)
```

Comment:
dont print :)

Reply:
ok, I guess I'm just using that to inspect the outputs/ debugging. Will not print anymore.